server/chat-commands/info.ts

```diff
@@ -2644,6 +2645,27 @@ export const commands: ChatCommands = {
 	},
 	showhelp: [`/show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`],
 
+	regtime: 'registertime',
+	async registertime(target, room, user) {
+		this.checkChat();
+		this.runBroadcast();
+		if (!user.autoconfirmed) return this.errorReply(`Only autoconfirmed users can use this command.`);
+		target = toID(target);
+		if (!target) target = user.id;
+		const raw = await Net(`https://${Config.routes.root}/users/${target}.json`).get().catch(() => {});
+		if (!raw) return this.errorReply(`An error occurred when retrieving user data.`);
+		// not in a try-catch block because if this doesn't work, this is a problem that should be known
+		const result = JSON.parse(raw);
+		const time = result.registertime * 1000;
+		const [day, minutes] = Chat.toTimestamp(new Date(time), {human: true}).split(' ');
+		return this.sendReplyBox(
+			`The user '${target}' registered ` +
+			`${Chat.toDurationString(Date.now() - time, {precision: 1})} ago, ` +
+			`on the date ${day}, at ${minutes}.`
+		);
```

Comment:
```suggestion
		const time = new Date(result.registertime * 1000);
		const regTime = Chat.toTimestamp(new Date(time), {human: true});
		const regTimeAgo = Chat.toDurationString(Date.now() - time, {precision: 1});
		this.sendReplyBox(
			`The user '${target}' registered ${regTimeAgo} ago, at ${regDate}.`
		);
```

Please don't use `return` on something just because it's the last function.

As for your `on, at` stuff, see design standard 1: less text is better.